cmd/gofish/search.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/fishworks/gofish"
+	"github.com/renstrom/fuzzysearch/fuzzy"
+	"github.com/spf13/cobra"
+)
+
+func newSearchCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "search",
+		Short: "perform a fuzzy search of fish food",
+		Run: func(cmd *cobra.Command, args []string) {
+			var foodNames = findFishFood()
+			var foundFood = make(map[string]bool)
+			if len(args) == 0 {
+				for _, found := range foodNames {
+					foundFood[found] = true
+				}
+			} else {
+				for _, keyword := range args {
+					for _, found := range fuzzy.Find(keyword, foodNames) {
+						foundFood[found] = true
+					}
+				}
+			}
+			var foodList []string
+			for food := range foundFood {
+				foodList = append(foodList, food)
+			}
+			fmt.Println(strings.Join(foodList, "\t\t\t"))
```

Comment:
`\n` instead of `\t`?

Reply:
The output mimics the same as `gofish list`: https://github.com/fishworks/gofish/blob/master/cmd/gofish/list.go

Then again, we probably need to reasonably split into a new line every few instances. My terminals all word wrap, but maybe someone else has word wrap disabled.

Reply:
My idea is that `\n` makes it possible to list fish name, fish version, and barrel from which it was installed!

Sample [scoop](https://github.com/lukesampson/scoop) output (`app-name app-version [bucket]`):
```
PS C:\Windows\system32> scoop list
Installed apps:

  cpu-z 1.84 [extras]
  losslesscut 1.10.0

PS C:\Windows\system32>
```

Reply:
That's not a bad idea! I'm off for the weekend but would you mind opening a new ticket with your thoughts? I have some ideas/thoughts around `gofish info` etc that may influence this :+1: